libbeat/cfgfile/registry.go

```diff
@@ -3,7 +3,7 @@ package cfgfile
 import "sync"
 
 type Registry struct {
-	sync.Mutex
+	sync.RWMutex
```

Comment:
I like that change. Could we move it to a separate PR to get it in quickly? Trying to reduce the size of this PR :-)

Reply:
huge :+1: on reducing the size of this PR.